test/functions/ruby-hello/handler.rb

```diff
@@ -1,7 +1,6 @@
-require 'json'
-
-def handler(event:, context:):
-    return {
-		statusCode: 200,
-        body: JSON.generate('Hello from ruby')
-    }
+def handler(event:, context:)
+  return {
+    statusCode: 200,
+    body: "Hello from ruby"
+  }
+end
```

Comment:
Why was this test changed?

Reply:
`JSON.generatel('Hello from ruby')` results in a string with escaped quotes, which isn't necessary here / is weird to add an assertion for.

Also, this makes the test (as written) pass.

```
$ irb
2.5.5 :001 > require 'json'
 => true 
2.5.5 :002 > JSON.generate('Hello from ruby')
 => "\"Hello from ruby\"" 
```

Reply:
I think the purpose was to test the `require 'json'` line. Can we do something that utilizes json serialization, maybe convert a ruby object to json?

Reply:
I have done this, resolving this convo